Data Structures and Algorithm (DSA)/readme.md

```diff
@@ -226,7 +233,93 @@ To the Point, Short and Practical - These 3 terms defines our Content
+| **Roadmap**            | [JavaScript DSA Roadmap](https://www.geeksforgeeks.org/javascript-data-structures-and-algorithms/) | Comprehensive guide to learning DSA in JavaScript.               |
+| **Array**              | [JavaScript Array Implementation](https://github.com/TheAlgorithms/Javascript/blob/master/Data-Structures/Arrays/array.js) | Example code for implementing arrays in JavaScript.              |
+|                        | [JavaScript Array Operations](https://www.geeksforgeeks.org/arrays-in-javascript/)                | Detailed explanations of array operations in JavaScript.         |
+| **String**             | [JavaScript String Implementation](https://github.com/TheAlgorithms/Javascript/blob/master/Data-Structures/Strings/string.js) | Example code for implementing strings in JavaScript.             |
+|                        | [JavaScript String Manipulations](https://www.geeksforgeeks.org/strings-in-javascript/)           | Guide on various string manipulations in JavaScript.             |
+| **Linked List**        | [JavaScript Linked List Implementation](https://github.com/TheAlgorithms/Javascript/blob/master/Data-Structures/LinkedList/linkedList.js) | Example code for implementing linked lists in JavaScript.        |
+|                        | [JavaScript Linked List Operations](https://www.geeksforgeeks.org/implementation-linkedlist-javascript/) | Detailed explanations of linked list operations in JavaScript.   |
+| **Tree**               | [JavaScript Tree Implementation](https://github.com/TheAlgorithms/Javascript/blob/master/Data-Structures/Trees/tree.js) | Example code for implementing trees in JavaScript.               |
+|                        | [JavaScript Tree Traversals](https://www.geeksforgeeks.org/tree-traversals-inorder-preorder-and-postorder/) | Detailed guide on tree traversal methods.                        |
+| **Graph**              | [JavaScript Graph Implementation](https://github.com/TheAlgorithms/Javascript/blob/master/Data-Structures/Graphs/graph.js) | Example code for implementing graphs in JavaScript.              |
+|                        | [JavaScript Graph Algorithms](https://www.geeksforgeeks.org/graph-data-structure-and-algorithms/) | Guide on various graph algorithms.                               |
+| **Sorting Algorithms** | [JavaScript Sorting Algorithms](https://github.com/TheAlgorithms/Javascript/tree/master/Sorting)  | Example code for different sorting algorithms in JavaScript.     |
+| **Search Algorithms**  | [JavaScript Search Algorithms](https://github.com/TheAlgorithms/Javascript/tree/master/Searches)  | Example code for different searching algorithms in JavaScript.   |
+| **GitHub Repository**  | [The Algorithms - JavaScript](https://github.com/TheAlgorithms/Javascript)                        | A collection of DSA implementations in JavaScript.               |
```

Comment:
Follow this format:
### Section name
<table width="100%">
 <tr>
   <th>Resource Name</th>
   <th>Description</th>
 </tr>
 <tr>
   <td><a href="URL HERE">RESOURCE NAME</a></td>
   <td>RESOURCE DESCRIPTION</td>
 </tr>
</table>